lib/push/creation.js

```diff
@@ -22,6 +22,7 @@ export function createEntities (context, entities, destinationEntities) {
       .then(partial(creationSuccessNotifier, operation))
       .catch(partial(handleCreationErrors, entity))
   }, {concurrency: 6})
+  .then((entries) => entries.filter((entry) => entry))
```

Comment:
Publishing does not handly empty entities. The `createEntries` function does also filter the output